samples/core/visualization/confusion_matrix_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .confusion_matrix import confusion_matrix_pipeline
+from ...test.util import run_pipeline_func, TestCase
+
+import kfp
+
+run_pipeline_func([TestCase(pipeline_func=confusion_matrix_pipeline,
+                            mode=kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE,
+                            arguments={
+                                kfp.dsl.ROOT_PARAMETER_NAME:
+                                'minio://mlpipeline/override/artifacts'
```

Comment:
Probably no need to test MinIO with every pipeline. We only need one example to verify MinIO works.

In the future, we can consider transparently changing pipeline root for all samples to GCS and MinIO. Test all the combinations.

Reply:
It makes sense, for now I am going to remove the MinIO root parameter, and verify the artifact name in testing.